documentation/API-onboarding.md

```diff
@@ -0,0 +1,75 @@
+
+- The API Backlog WG coordinator checks that the template is duly filled in. Otherwise, the API proposal owner will be requested to provide missing information.
+- After this sanity check, each WG participant declares his/her support (which is also documented in the template).
+
+Finally the pull request is merged into the main branch and the API proposal template is sent to the Steering Committee of CAMARA at least one week prior to the Steering Committee meeting where the API proposal shall be voted upon.
+
+This whole step shall be done within 2 regular meetings of the API Backlog WG. In parallel, the API proposal is formally registered in CAMARA by adding it to the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md) (API Backlog WG creates a new line in the table and fills in API family name, API family owner, API family proposal registration date, the link to the application template and the supporters for it).
+
+## API proposal Steering Committee decision
+
+Upon receiving the API proposal and notification from the API Backlog WG, the Steering Committee studies the proposal and votes it at the Steering Committee meeting.
+
+- No-Go! Objecting companies shall provide justifications why. The API proposal is rejected and will not be included in any API Sub Project.
+- Go! In this case, the Steering Committee shall specify whether the API proposal is to be hosted by a new or existing Sub Project.
+
+Main tenets for Steering Committee voting:
+
+- Tenet #1: Open-source projects are contribution-driven. There should be no guidelines to prevent partners to contribute. If a company would like to work on a certain API proposal they should be welcome to do it.
+- Tenet #2: Steering Committee shall make sure that APIs belonging together (close topic/content, API families) are developed within the same Sub Project, while ensuring that Sub Projects are not too small (difficult to track them down) or too big (difficult to be managed).
+- Tenet #3: There could be situations where API proposal complies with acceptance criteria (template duly filled out) but has not supporters other than the proponent company. What to do in this situation? Recommendation is to accept the API proposal and see how successful it will be in terms of industry adoption.
+
+The Steering Committee documents the decision in the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md) (fills in columns Steering Committee date and Steering Committee decision / Sub Project; in case of a No-Go "Rejected" is documented there).
+
+## API management and maintenance
+
+In case of a GO decision of the Steering Committee the named Sub Project takes over responsibility for the new API or API family and starts making contributions for the API definition(s), API documentation, provider implementation and test artefacts.
+
```

Comment:
IS there a timeframe on how quick the sub project should be created e.g. some sort of SLA so we can start contributions ASAP

Reply:
@nathanrader : that should be covered by the release guidelines from commonalities